Xamarin.Forms.Platform.iOS/CollectionView/TemplatedCell.cs

```diff
@@ -69,5 +69,21 @@ void ClearSubviews()
 				ContentView.Subviews[n].RemoveFromSuperview();
 			}
 		}
+
+		public override bool Selected
+		{
+			get => base.Selected;
+			set
+			{
+				base.Selected = value;
+
+				var element = VisualElementRenderer?.Element;
+
+				if (element != null)
+				{
+					VisualStateManager.GoToState(VisualElementRenderer.Element, value ? "Selected" : "Normal");
```

Comment:
Probably better to use the variable. And, we probably need to start thinking about these magic strings - they should be constants.

```suggestion
					VisualStateManager.GoToState(element, value ? "Selected" : "Normal");
```